src/app/globals.css

```
@@ -5,6 +5,11 @@
 :root {
   --font-span: "span", serif;
   --bw: 1440;
+
+  .swiper-slide-event-detail {
+    width: calc((325 / 1440) * 100 * 1vw) !important;
+    height: 60vh !important;
```

ini juga bisa di calculate kalau mau, tapi kalau 60vh udah works,  di semua viewport aman sih frans

```suggestion
   --bh: 810;
    height: calc((60/var(--bh)*100vh)) !important;
```